webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -4019,7 +4020,16 @@ var userOverride = {
                             if (MCK_TRIGGER_MSG_NOTIFICATION_TIMEOUT > 0) {
                                 ALStorage.clearMckMessageArray();
                             }
-                            $applozic.fn.applozic('loadTab', null, callback);
+                            if (kommunicate._globals.zendeskChatSdkKey) {
+                                var groupId = result.response[result.response.length-1].id;
```

Comment:
If this is a chat server api result then we can add checks because there has been some cases where the api responded with 200 ok status but the response was undefined

Reply:
Added null check.